src/Spec.ts

```diff
@@ -1438,6 +1438,41 @@ ${this.opts.multipage ? `<li><span>Navigate to/from multipage</span><code>m</cod
         this.doc.body.insertBefore(h1, this.doc.body.firstChild);
       }
     }
+
+    // opengraph tags
+    const metas = [...this.doc.querySelectorAll('meta')];
+    const insertMetaTag =
+      metas.length === 0
+        ? (node: HTMLMetaElement) => this.doc.head.prepend(node)
+        : (node: HTMLMetaElement) => metas[metas.length - 1].after(node);
+    if (!metas.some(n => n.getAttribute('property') === 'og:description')) {
+      const description =
+        this.opts.description ??
+        (this.doc.querySelector('emu-intro') ?? this.doc.querySelector('emu-clause'))?.textContent
+          ?.slice(0, 300)
+          .trim();
```

Comment:
```suggestion
        (this.opts.description ?? (this.doc.querySelector('emu-intro') ?? this.doc.querySelector('emu-clause'))?.textContent)
          ?.slice(0, 300)
          .trim();
```

Maybe also slice the explicit description?